rdflib/graph.py

```diff
@@ -389,6 +389,27 @@ def add(self, triple):
         assert isinstance(o, Node), "Object %s must be an rdflib term" % (o,)
         self.__store.add((s, p, o), self, quoted=False)
 
+    def add_all(self , triples, all_or_none=True):
```

Comment:
I know the name `add_all` is more pythonic and PEP8-compliant than `addAll`, but the rest of the methods on `Graph` were written a very long time before any sense of style existed in python. One day we might break everyones code by renaming them all to PEP8-compant names, but until then we'd better stick with the existing naming conventions in this file, rather than introduce inconsistencies.

Reply:
I gave the name following PEP8 standards and also the current method names in `graph.py` namely `subject_predicates`, `subject_objects`, `all_nodes`. Should I still change the name to `addAll`?

Reply:
I'll have to get opinion of the other rdflib maintainers on that.

Reply:
Okay sure, do let me know.